### DISCUSSION TITLE
Adding an icon next to a title for a code block.

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [squidfunk/mkdocs-material#3254] [Q&A] [answered]
I am trying to add an icon (provided by the `pymdownx.emoji` extension) next to a code block title. As an example, immediate left of `bubble_sort.py` in **Results**. An example octicon `:octicons-file-code-16:`

But no matter what combinations I try (placing it before `=`, after `=`, inside the title), it seems to break the parsing and code block. 
Any suggestions ?

![image](https://user-images.githubusercontent.com/34304644/143417218-8f217bc5-6f8d-4758-a56a-c2bb54b24f61.png)

**Comment by squidfunk** [comment, marked as answer]
See both comments:

- https://github.com/squidfunk/mkdocs-material/issues/2997#issuecomment-955177219
- https://github.com/squidfunk/mkdocs-material/issues/2997#issuecomment-955209436

**Comment by UltraInstinct05** [comment]
Okay. From the comments, I see that it's a non trivial feature to add. Thanks !

Edit: You mentioned some custom CSS can be used to do it. Any examples you can link if possible ?

**Reply by squidfunk** [reply]
https://facelessuser.github.io/pymdown-extensions/extensions/superfences/#code-block-title-headers has some examples.
